manager/integration/tests/test_basic.py

```diff
@@ -59,6 +63,44 @@ def test_hosts_and_settings(clients):  # NOQA
         assert setting["value"] == old_value
 
 
+def volume_read(fdev, start, count):
+    fdev.seek(start)
+    return fdev.read(count)
+
+
+def volume_write(fdev, start, data):
+    fdev.seek(start)
+    fdev.write(data)
+    fdev.flush()
+
+
+def volume_valid(dev):
+    return stat.S_ISBLK(os.stat(dev).st_mode)
+
+
+def volume_rw_test(dev, count):
```

Comment:
Just make the function that verifying the consistency of device at `loc` for `size`. Don't make caller compare check_count.

And put related functions in the `common.py`